chapter2/tests/t13/Dockerfile

```diff
@@ -1,17 +1,20 @@
+ARG NODE_VERSION=14-slim
+ARG NGINX_VERSION=1.22.0-alpine
+
 # syntax=docker/dockerfile:1.2 
-FROM --platform=${BUILDPLATFORM:-linux/amd64} node:14-slim AS builder
+FROM --platform=${BUILDPLATFORM:-linux/amd64} node:"${NODE_VERSION}" AS builder
 
 RUN apt update
 RUN apt install git -y
 
 WORKDIR /opt/flatris
 RUN git clone https://github.com/rtest12/flatris.git /opt/flatris
 
-RUN yarn install --arch=$BUILDPLATFORM
-RUN yarn test
-RUN yarn build
+RUN yarn install --arch=$BUILDPLATFORM \
+  && yarn test \
+  && yarn build
 
-FROM --platform=${BUILDPLATFORM:-linux/amd64} nginx:1.22.0-alpine AS runtime
+FROM --platform=${BUILDPLATFORM:-linux/amd64} nginx:"${NGINX_VERSION}" AS runtime
```

Comment:
The use of nginx functions is not implemented in any way in the current task

Reply:
> The use of nginx functions is not implemented in any way in the current task

@udaltsovra
Yes, you're right, I was inattentive and now switched to Alpine. For some reason, the final image size increased even though I built it with --no-cache. Strange.
Done